adapters/onebot/src/bot/message.ts

```diff
@@ -141,8 +153,10 @@ export class OneBotMessenger extends Messenger<BaseBot> {
         await this.flush()
         this.stack.shift()
         await this.forward()
+      } else if ('id' in attrs) {
+        this.stack[0].data = attrs.id.toString()
       } else {
-        Object.assign(this.stack[0].author, pick(attrs, ['userId', 'username', 'nickname', 'avatar']))
+        Object.assign(this.stack[0].data, pick(attrs, ['userId', 'user-id', 'username', 'nickname', 'time']))
```

Comment:
Why removed `avatar`?

Reply:
because go-cqhttp didn't provide api to set avatar:
![image](https://user-images.githubusercontent.com/64234553/217773042-11309bbe-18ad-45af-aa9f-ffaa4099239d.png)
if you want set message avatar, just need to set `userId` .

Reply:
Ah ok.